### PR TITLE
feat: add the option to inject binary libraries to install in Dockerfile

### PR DESCRIPTION
**Pull request by SdgJlbl**
Fixes FL-1741

### CODE REVIEW

File: tests/remote/register/test_register.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def test_get_base_docker_image_gpu():
     && apt-get install -y software-properties-common\
     && add-apt-repository -y ppa:deadsnakes/ppa\
     && apt-get -y upgrade\
-    && apt-get install -y python3.11 python3.11-venv python3-pip\
+    && apt-get install -y python3.11 python3.11-venv python3-pip \
```
Comment (ThibaultFy) [inline tests/remote/register/test_register.py:67]:
```suggestion
    && apt-get install -y python3.11 python3.11-venv python3-pip\
```

Comment (SdgJlbl) [reply tests/remote/register/test_register.py:67]: This whitespace is required to make the test pass, actually :)

Comment (ThibaultFy) [reply tests/remote/register/test_register.py:67]: 🤔  weird

Comment (SdgJlbl) [reply tests/remote/register/test_register.py:67]: That's the place where the binary dependencies would be if there were defined

Comment (ThibaultFy) [reply tests/remote/register/test_register.py:67]: ah ok make sense,

**Review by ThibaultFy — APPROVED** [review]
Lgtm thanks a lot :)

**Pull request merged**